nautobot/core/api/serializers.py

```diff
@@ -134,6 +134,7 @@ class BaseModelSerializer(OptInFieldsMixin, serializers.HyperlinkedModelSerializ
 
     serializer_related_field = NautobotHyperlinkedRelatedField
 
+    id = serializers.UUIDField(read_only=False, default=serializers.CreateOnlyDefault(uuid.uuid4))
```

Comment:
I believe the test failure is because Django's built-in Group model has an integer PK instead of a UUID PK. Might just need to override this field on `GroupSerializer` (and probably also `ContentTypeSerializer` for similar reasons).

Reply:
We have success! Thanks for the help here @glennmatthews.